r2/r2/controllers/listingcontroller.py

```diff
@@ -977,6 +977,29 @@ def GET_trophies(self, user):
             return self.abort404()
         return self.api_wrapper(get_usertrophies(user))
 
+    @validate(user=VExistingUname('username'))
+    @api_doc(section=api_section.users, uri='/user/{username}/moderated')
+    def GET_moderated_srs(self, user):
+        """Gets a list of subreddits moderated by this user."""
+        if not is_api():
+            return self.abort404()
+
+        if not user:
+            return self.abort404()
+
+        mod_sr_ids = Subreddit.reverse_moderator_ids(user)
+        all_mod_srs = Subreddit._byID(mod_sr_ids, data=True,
+                                      return_dict=False, stale=True)
+        mod_srs = [sr for sr in all_mod_srs if sr.can_view_in_modlist(c.user)]
+
+        if not mod_srs:
+            return self.api_wrapper([])
+
+        templ = TrimmedSubredditJsonTemplate()
+        resp = [templ.render(sr).finalize() for sr in mod_srs]
+
```

Comment:
@Pokechu22 Forgive my ignorance if this is incorrect as I haven't tested your branch on my local install; but I was playing around with templates just now; and the values for the dictionary keys came out in unicode (`u'text here' format`); so you may have that problem as well (unless `api_wrapper()` takes care of that?)